subsys/net/openthread/rpc/common/ot_rpc_ids.h

```diff
@@ -6,15 +6,15 @@
 
 /** @brief Command IDs accepted by the OpenThread over RPC client.
  */
-enum ot_rpc_cmd_client {
+enum OT_RPC_CMD__rpc_cmd_client {
```

Comment:
This doesn't seem to follow naming conventions used elsewhere in NCS. Why this change?

Reply:
I don't recall how it got there. Never the less it's fixed.